### PR TITLE
[cypress] + com_cache

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
add a simple test for the view `/administrator/index.php?option=com_cache&view=cache`

### Testing Instructions
run
`npm run cypress:run -- --spec tests/System/integration/administrator/components/com_cache/Default.cy.js
`
### Actual result BEFORE applying this Pull Request
N/A

### Expected result AFTER applying this Pull Request

a test for the view

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: tests/System/integration/administrator/components/com_cache/Default.cy.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    cy.get('div.alert.alert-info').should('contain.text', 'Select the Clear Expired Cache button');
+  });
+
+  it('can display a list of chached items', () => {
+    cy.get('tr.row0').should('contain.text', '_media_version');
+  });
+
+  it('can clear expired cache', () => {
+    cy.get('#toolbar-delete2').click();
+    cy.on('window:confirm', () => true);
```
Comment (muhme) [inline tests/System/integration/administrator/components/com_cache/Default.cy.js:21]: This test spec works in dev-4.4 for me and starting with 5.1-dev the following error occurs:
```
  1) Test in backend that the cache
       can clear expired cache:
     AssertionError: Timed out retrying after 4000ms: Expected to find content: 'Expired cached items have been cleared' within the element: <div#system-message-container> but never did.
      at Context.eval (webpack://joomla/./tests/System/integration/administrator/components/com_cache/Default.cy.js:22:91)
```
Reason is a new changed confirmation popup:

warning

I would recommend to make the test code release-independent to not maintain two different versions. I tested the following code change with all four branches successfully:

**Suggestion:**
```suggestion
    cy.on('window:confirm', () => true);
    // Starting with Joomla 5.1 check if the 'Yes' button exists and click it if present
    cy.get('body').then(($body) => {
      if ($body.find('div.buttons-holder button[data-button-ok]').length > 0) {
        cy.get('div.buttons-holder button[data-button-ok]').click();
      }
    });

File: tests/System/integration/administrator/components/com_cache/Default.cy.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    // Starting with Joomla 5.1 check if the 'Yes' button exists and click it if present
```
Comment (laoneo) [inline tests/System/integration/administrator/components/com_cache/Default.cy.js]: We should not add Joomla 5 related code in 4.4. Better would be to make then two separate pr's and the release managers have to deal then with the merge conflicts.

Comment (alikon) [reply tests/System/integration/administrator/components/com_cache/Default.cy.js]: ok for 5 i'll do a new pr

**Comment by muhme** [comment]
✅ successfully tested this additional test spec on the branches dev-4.4, dev-5.1, dev-5.2 and dev-6.0, as single spec and within the full System Test

File: tests/System/integration/administrator/components/com_cache/Default.cy.js (hunk 2 above)
Comment (muhme) [reply tests/System/integration/administrator/components/com_cache/Default.cy.js]: @laoneo All tests are currently carried out with the same code in all four development branches. Is it really worth making a difference? It checks that the button exists, so it doesn't hurt, does it?

Comment (muhme) [reply tests/System/integration/administrator/components/com_cache/Default.cy.js]: > @laoneo All tests are currently carried out with the same code in all four development branches. Is it really worth making a difference? It checks that the button exists, so it doesn't hurt, does it?

@laoneo No, i am wrong. There are already differences, e.g. `administrator/components/com_cpanel/CPanel.cy.js` exists in 5.1-dev and following, but not in 4.4-dev. Therefore, i will remember that System Tests are release-dependent.

**Comment by laoneo** [comment]
Thanks!
